index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="src/styles.css">
+    <title>Hungry Caterpillar</title>
+</head>
+<body>
+    <header>
+        <h1>Hungry Caterpillar</h1>
+    </header>
+    <main>
+        <section class="buttons">
+            <h2>Feed me fruits!</h2>
+            <button class="fruit" value="apple"><img src="assets/apple.jpeg" alt="apple"></button>
```

Comment:
For better styling/organization and to make things easier to read lay out nested tags like so:
`
<button class="fruit" value="apple">
        <img src="assets/apple.jpeg" alt="apple">
</button>
`